setup.py

```diff
@@ -50,9 +50,16 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        # TODO(HideakiImamura) Unpin mypy version after fixing "Duplicate modules" error in
-        # tutorials.
-        "checking": ["black", "hacking", "isort", "mypy==0.790", "blackdoc"],
+        "checking": [
+            "black",
+            "hacking",
+            "isort",
+            "blackdoc",
+            "mypy",
+            "types-setuptools",
+            "types-redis",
+            "types-PyYAML",
```

Comment:
:+1:

Reply:
Thanks! I confirmed that adding `types-setuptools`, `types-redis` and `types-PyYAML` is required.
The following error will occur without them.
```console
$ mypy .
setup.py:6: error: Library stubs not installed for "pkg_resources" (or incompatible with Python 3.9)
setup.py:6: note: Hint: "python3 -m pip install types-setuptools"
optuna/storages/_redis.py:30: error: Library stubs not installed for "redis" (or incompatible with Python 3.9)
optuna/storages/_redis.py:30: note: Hint: "python3 -m pip install types-redis"
optuna/storages/_redis.py:30: note: (or run "mypy --install-types" to install all missing stub packages)
optuna/storages/_redis.py:30: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
optuna/cli.py:30: error: Library stubs not installed for "yaml" (or incompatible with Python 3.9)
optuna/cli.py:30: note: Hint: "python3 -m pip install types-PyYAML"
tests/test_cli.py:16: error: Library stubs not installed for "yaml" (or incompatible with Python 3.9)
Found 4 errors in 4 files (checked 259 source files
```